.circleci/config.yml

```diff
@@ -446,7 +455,7 @@ workflows:
             - setup
           filters:
             tags:
-              only: /.*/
+              only: /v[0-9]+(\.[0-9]+)*(\-snapshot)?(\-hotfix(\.[0-9]+))?(\-pisp)/
```

Comment:
the last section (\-pisp) should be optional not mandatory

Reply:
Can you explain your reasoning? I thought we were denoting builds with `-pisp`. I might be misunderstanding something.

Reply:
@kleyow the filter tag rule will be triggered only if there is `-pisp` postfix because this section in mandatory in regexp tag, so it is blocking triggering for other tags.

Reply:
I suggest using `(\-pisp)?` expression instead

Reply:
done.